src/lockfile/parse.js

```diff
@@ -155,7 +159,7 @@ function* tokenise(input: string): Iterator<Token> {
     }
 
     col += chop;
-    lastNewline = input[0] === '\n';
+    lastNewline = input[0] === '\n' || input[1] === '\n';
```

Comment:
does input[0] matter when input[1] === '\n'?

Reply:
I guess I can change this to `input[0] === '\n' || (input[0] === '\r' && input[1] === '\n')` to be precise?